code/modules/clothing/suits/miscellaneous.dm

```diff
@@ -459,3 +459,40 @@
 	)
 	body_parts_covered = UPPER_TORSO|ARMS
 	price_tag = 600
+
+/obj/item/clothing/suit/suicidevest
+	name = "bomb vest"
+	desc = "A mess of wire and unstable plasma crystals, fashioned into a bomb. Designed to be strapped to the torso."
+	description_info = "Can be detonated in the Object tab."
+	icon_state = "bombvest"
+	item_state = "bombvest"
+	armor = list(
+		melee = 10,
+		bullet = 5,
+		energy = 5,
+		bomb = 0,
+		bio = 0,
+		rad = 0
+	)
+	spawn_blacklisted = TRUE
+
```

Comment:
```suggestion
	style=STYLE_LOW

```
Specify the style to avoid retrieving STYLE_HIGH from clothing/suit. It can be STYLE_NONE or STYLE_LOW. I can concede that wearing a suicide vest is kinda stylish and swaggy but not to the point of being high style.